platform/vechain/api.go

```diff
@@ -91,44 +91,48 @@ func GetAddressTransactions(address string, token string) []models.Tx {
 	return txsNormalized
 }
 
-func Normalize(tr *Tx, receipt *TxReceipt, output *TxReceiptOutput, address string, token string) (models.Tx, bool) {
+func Normalize(tr *Tx, receipt *TxReceipt, output *TxReceiptOutput, address string, token string) (tx models.Tx, ok bool) {
 	transferType, _ := transferType(*output)
+	var timestamp = tr.Meta.BlockTimestamp
 	transfer := output.Transfers[0]
 	sender := transfer.Sender
 	recipient := transfer.Recipient
 
-	if transferType == models.TxTransfer && (sender == address || recipient == address) { // Currently supports only transfer transactions
-		var from  = sender
-		var to = recipient
-		
-		var fee, value string
-		if token == VeThorContract {
-			fee = "0"
-			value = hexaToIntegerString(receipt.Paid)
-		} else {
-			fee = hexaToIntegerString(receipt.Paid)
-			value = hexaToIntegerString(output.Transfers[0].Amount)
+	tx = models.Tx{
+		ID:       tr.Meta.TxID,
+		Coin:     coin.VET,
+		From:     sender,
+		To:       recipient,
+		Date:     timestamp,
+		Type:     transferType,
+		Block:    tr.Meta.BlockNumber,
+		Sequence: uint64(timestamp),
```

Comment:
iirc, we chose to omit `Sequence` for coins that don't have tx sequences/nonces, so you can remove this